README.md

```diff
@@ -158,22 +103,140 @@ More information about the
+| triplestore.baseUrl | Base URL for the triplestore | http://localhost:8080/fuseki/test/update | 
+| triplestore.input.stream |   The JMS topic or queue serving as the message source    | broker:topic:fedora | | 
+| triplestore.reindex.stream |   The JMS topic or queue serving as the reindex message source    | broker:queue:solr.reindex | | 
+| triplestore.indexing.predicate |   ?    | false | | 
+| triplestore.filter.containers |   A comma-separate list of containers that should be ignored by the indexer  | http://localhost:8080/fcrepo/rest/audit | | 
+| triplestore.namedGraph |  ?  | null | | 
+| triplestore.prefer.include |  ?  | null | | 
+| triplestore.prefer.omit |  ?  | http://www.w3.org/ns/ldp#PreferContainment | | 
+
+### LDPath Service
+
+This application implements an LDPath service on repository
+resources. This allows users to dereference and follow URI
+links to arbitrary lengths. Retrieved triples are cached locally
+for a specified period of time.
+
+More information about LDPath can be found at the [Marmotta website](http://marmotta.apache.org/ldpath/language.html).
+
+Note: The LDPath service requires an LDCache backend, such as `fcrepo-service-ldcache-file`.
+
+#### Usage
+The LDPath service responds to `GET` and `POST` requests using any accessible resources as a context.
+
+For example, a request to
+`http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object`
+will apply the appropriate ldpath program to the specified resource. Note: it is possible to
+identify non-Fedora resources in the context parameter.
+
+A `GET` request can include a `ldpath` parameter, pointing to the URL location of an LDPath program:
+
+    `curl http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object&ldpath=http://example.org/ldpath`
+
+Otherwise, it will use a simple default ldpath program.
+
+A `POST` request can also be accepted by this endpoint. The body of a `POST` request should contain
+the entire `LDPath` program. The `Content-Type` of the request should be either `text/plain` or
+`application/ldpath`.
+
+    `curl -XPOST -H"Content-Type: application/ldpath" -d @program.txt http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object
+
+#### Properties
+| Name      | Description| Default Value | Values |
+| :---      | :---| :----   | --- |
+| fcrepo.cache.timeout | The timeout in seconds for the ldpath cache | 0 | | 
+| rest.prefix | The LDPath rest endpoint prefix |  no | /ldpath| |
+| rest.port| The LDPath rest endpoint port |  no | 9085 |
+| rest.host| The LDPath rest endpoint host |  no | localhost |
+| cache.timeout | LDCache ?  timeout in seconds  |  no | 86400  | |
+| ldcache.directory | LDCache directory  |  no | ldcache/  | |
+| ldpath.transform.path | The LDPath transform file path | classpath:org/fcrepo/camel/ldpath/default.ldpath | For local file paths use `file:` prefix. e.g `file:/path/to/your/transform.txt` |
+
+### Reindexing Service
```

Comment:
There's already a "Repository Re-Indexer" section lower in the document which is a duplicate of this section.

Reply:
Fixed; removed the lower section and consolidated into the one above